docs/build/security-docs/threat-modeling/pizza-restaurant-example.mdx

```diff
@@ -0,0 +1,76 @@
+
+1. The customer builds their pizza on the website from a list of ingredients.
+2. When finished building, the website calculates the price of the pizza by querying the database.
+3. The customer enters their payment information on the website and the website connects to the credit card processor to process the payment.
+4. The pizza order is added to the database.
+5. For the social component, once the order is in the database, it is also displayed on the starting page of the webpage with the user’s username.
+6. Pizza makers in the pizza shop check the database and begin building the pizza through various stations, including dough, sauce, cheese, toppings, baking, and boxing. One pizza worker is at each station.
+7. At each step, the pizza maker updates the database with current progress. As the database is updated, the pizza tracker on the website is updated, showing current progress and who has completed that step.
+8. Customer picks up their pizza once it is ready.
+
+![Threat Modeling](/assets/security/threat-modeling.png)
+
+## What can go wrong?
+
+### STRIDE reminders
+
+| Mnemonic Threat | Definition | Question |
+| --- | --- | --- |
+| **S**poofing | The ability to impersonate another user or system component to gain unauthorized access. | Is the user who they say they are? |
+| **T**ampering | Unauthorized alteration of data or code. | Has the data or code been modified in some way? |
+| **R**epudiation | The ability for a system or user to deny having taken a certain action. | Is there enough data to “prove” the user took the action if they were to deny it? |
+| **I**nformation Disclosure | The over-sharing of data expected to be kept private. | Is there anywhere where excessive data is being shared or controls are not properly in place to protect private information? |
+| **D**enial of Service | The ability for an attacker to negatively affect the availability of a system. | Can someone, without authorization, impact the availability of the service or business? |
+| **E**levation of Privilege | The ability for an attacker to gain additional privileges and roles beyond what they initially were granted. | Are there ways for a user, without proper authentication (verifying identity) and authorization (verifying permission) to gain access to additional privileges, either through standard (normally legitimate) or illegitimate means? |
+
+### Threat table
+
+| Threat | Issues |
+| --- | --- |
+| **S**poofing | **Spoof.1** - (Step 1) An attacker could submit an order as another user. <br /><br /> **Spoof.2** - (Step 3) If credit card information is stored in the user profile, a spoofed pizza order could be submitted automatically to the CC processor, making another user pay for the pizza order. <br /><br /> **Spoof.3** - (Step 7) An attacker “blasts” erroneous pizza updates at the website, eroding trust in the data and process. |
```

Comment:
nothing to add here, just that i'm with you: i **super** wish there were a better way to put line breaks into markdown tables. 😢